src/login/logind-dbus.c

```diff
@@ -2945,6 +2946,16 @@ int manager_start_scope(
                         return r;
         }
 
+        if (!isempty(requires)) {
+                r = sd_bus_message_append(m, "(sv)", "After", "as", 1, requires);
+                if (r < 0)
+                        return r;
+
+                r = sd_bus_message_append(m, "(sv)", "Requires", "as", 1, requires);
+                if (r < 0)
+                        return r;
+        }
```

Comment:
> not Wants=

because if one stops user@.service, we should terminate user's sessions too in order to avoid consistency problems (dangling references to the bus, other services etc. — people are not prohibited to run vital parts of their sessions inside the user manager).

> or BindsTo=

TBH I don't understand the practical difference between these two dependency types.